modules/utils/src/logger.ts

```diff
@@ -3,12 +3,12 @@ import { ILogger, ILoggerService } from "@connext/types";
 export const logTime = (log: ILogger, start: number, msg: string) => {
   const diff = Date.now() - start;
   const message = `${msg} in ${diff} ms`;
-  if (diff < 10) {
-    log.info(message);
+  if (diff < 25) {
+    log.debug(message);
```

Comment:
Can we keep this as info? There's a few logs in cf-core that are being made with this which we lose with debug level.

Reply:
Specifically I'm talking about in protocols, we log things like `... Started` but the `... Complete` log is in this logTime function so we would lose it which makes debugging very difficult.